config/src/test/java/org/hyperledger/besu/config/GenesisReaderTest.java

```diff
@@ -41,11 +41,12 @@ public void readGenesisFromObjectNode() {
     final var configNode = mapper.createObjectNode();
     configNode.put("londonBlock", 1);
     final var allocNode = mapper.createObjectNode();
-    allocNode.put(Address.BLS12_G2MULTIEXP.toUnprefixedHexString(), generateAllocation(Wei.ONE));
+    allocNode.putIfAbsent(
```

Comment:
Why the change from put to putIfAbsent? I guess either should be fine since the object node is being created in the test

Reply:
put with those args (String, JsonNode) is deprecated. putIfAbsent was an easy fix and yeah - it's a test, and the test still passes, and I didn't think any further about it